features/datachoices/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -66,6 +66,8 @@
 
     <bean id="dataSourceFactoryBean" class="org.opennms.core.db.DataSourceFactoryBean" />
 
+    <reference id="flowQueryService" interface="org.opennms.netmgt.flows.api.FlowQueryService" availability="mandatory"/>
```

Comment:
I think we shouldn't depend on flow service to be always available, so remove `mandatory`